YAMS-LIB/Program.cs

```diff
@@ -2146,12 +2162,13 @@ void RotateTextureAndSaveToTexturePage(int rotation, UndertaleTexturePageItem te
         ReplaceGMLInCode(eTankCharacterEvent, "global.maxhealth += (100 * oControl.mod_etankhealthmult)", $"global.maxhealth += {seedObject.Patches.EnergyPerTank}");
 
         // Flashlight
-        PrependGMLInCode(characterVarsCode, "global.flashlightLevel = 0;");
+        PrependGMLInCode(characterVarsCode, "global.flashlight = 0;");
+        PrependGMLInCode(characterVarsCode, "global.blindfold = 0;");
         ReplaceGMLInCode(gmData.Code.ByName("gml_Script_ApplyLightPreset"), "global.darkness", "lightLevel");
         PrependGMLInCode(gmData.Code.ByName("gml_Script_ApplyLightPreset"),
             """
             var lightLevel = 0
-            lightLevel = global.darkness - global.flashlightLevel
+            lightLevel = global.darkness - (global.flashlight - global.blindfold)
```

Comment:
This logic seems overly complicated.
Why not do:
1. Keep flashlightLevel
2. Make flashlight items increase flashLightLevel based on count
3. Make blindfold items decrease flashlightLevel based on count
?